### DISCUSSION TITLE
how can i Automatically import packages in golang

### DISCUSSION THREAD

**Discussion by codeErrorSleep** [helix-editor/helix#7610] [Q&A] [answered]
My language is golang. When I copy the code from other places, I find that it cannot automatically import the required packages.

My helix version is 23.05

I have installed gopls
![image](https://github.com/helix-editor/helix/assets/20470033/c60583f3-ae36-4c86-bd77-138831759354)

just like that
![image](https://github.com/helix-editor/helix/assets/20470033/fc7de4b6-c210-406a-aa19-1af69c862b61)

**Comment by codeErrorSleep** [comment]
my languages.toml
i have copy from https://github.com/helix-editor/helix/blob/master/languages.toml
![image](https://github.com/helix-editor/helix/assets/20470033/ef9573b0-11a7-4207-989d-da34cc3ef38a)

**Comment by notmgsk** [comment, marked as answer]
`space a` -> `Organize imports` should add the necessary imports.

**Reply by codeErrorSleep** [reply]
it is work,Thanks bro.
